projects/components/src/utils/test/widget-object/angular-widget-object.ts

```diff
@@ -208,6 +208,32 @@ export class TestElement implements Iterable<TestElement> {
         return this.elements[0].injector;
     }
 
+    /**
+     * Private helper function to find the first parent element that matches css selector
```

Comment:
Please remove the words private helper function, they are already all implied/expressed in code.

```
Finds the first parent element that matches the CSS selector
```